extensions/ql-vscode/src/vscode-tests/minimal-workspace/databases.test.ts

```diff
@@ -191,74 +176,74 @@ describe("databases", () => {
         item: undefined,
         kind: DatabaseEventKind.Add,
       };
-      expect(spy).to.have.been.calledWith(mockEvent);
+      expect(onDidChangeDatabaseItem).toBeCalledWith(mockEvent);
     });
 
-    it("should add a database item source archive", async function () {
+    it("should add a database item source archive", async () => {
       const mockDbItem = createMockDB();
       mockDbItem.name = "xxx";
-      await (databaseManager as any).addDatabaseSourceArchiveFolder(mockDbItem);
+      await databaseManager.addDatabaseSourceArchiveFolder(mockDbItem);
 
       // workspace folders should be updated. We can only check the mocks since
       // when running as a test, we are not allowed to update the workspace folders
-      expect(workspace.updateWorkspaceFolders).to.have.been.calledWith(1, 0, {
+      expect(workspace.updateWorkspaceFolders).toHaveBeenCalledWith(1, 0, {
         name: "[xxx source archive]",
         // must use a matcher here since vscode URIs with the same path
         // are not always equal due to internal state.
-        uri: sinon.match.has(
-          "fsPath",
-          encodeArchiveBasePath(sourceLocationUri().fsPath).fsPath,
-        ),
+        uri: expect.objectContaining({
+          fsPath: encodeArchiveBasePath(sourceLocationUri().fsPath).fsPath,
+        }),
       });
     });
 
     it("should remove a database item", async () => {
       const mockDbItem = createMockDB();
-      sandbox.stub(fs, "remove").resolves();
+      const removeMock = jest.spyOn(fs, "remove");
+      removeMock.mockImplementation(() => Promise.resolve());
```

Comment:
This can be combined in a single line, `mockImplementation` will return the spy. You can also use `mockResolvedValue` instead of `mockImplementation`.